.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "black"
+}
```

Comment:
🔧 Can you add .vscode the the .gitignore file? That way this IDE-specific content won't be commit to the repo

Reply:
Done!